sanic/server.py

```diff
@@ -167,12 +167,18 @@ def on_message_complete(self):
     # -------------------------------------------- #
 
     def write_response(self, response):
+        keep_alive = (
```

Comment:
Why not leave this in the try/except? It seems that would be more useful in case the parser object was invalid (e.g. the current pipelining response)